### PR TITLE
HOS-82: Animal list page

### PR DESCRIPTION
**Pull request by yana617**

### CODE REVIEW

File: src/components/animal-list-view/Search.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Search',
+  data() {
+    return { searchText: '' };
+  },
+  computed: {},
```
[the remark this answers is not in the document]
Comment (Aphelmion) [reply src/components/animal-list-view/Search.vue:15]: molodec

File: package.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
     "ant-design-vue": "2.2.8",
     "axios": "0.27.2",
     "core-js": "3.31.0",
+    "dayjs": "^1.11.5",
```
Comment (yana617) [inline package.json:24]: version

File: src/components/animal-list-view/AnimalCard.vue [hunk 3]
```diff
@@ -0,0 +1,138 @@
+<template>
+  <router-link class="animal-card" :to="`/animals/${id}`">
+    <div class="animal-card__image-container">
+      <img class="animal-card__image" alt="avatar" :src="photos[0]" />
```
Comment (yana617) [inline src/components/animal-list-view/AnimalCard.vue:4]: default image case

Comment (yana617) [reply src/components/animal-list-view/AnimalCard.vue:4]: after design

**Pull request merged**